src/include/distributed/metadata_utility.h

```diff
@@ -259,14 +259,17 @@ extern void CreateTruncateTrigger(Oid relationId);
 extern TableConversionReturn * UndistributeTable(TableConversionParameters *params);
 
 extern void EnsureDependenciesExistOnAllNodes(const ObjectAddress *target);
```

Comment:
EnsureDependenciesExistOnAllNodes and all other internal functions should become `static` functions defined locally in C files. We should only expose the APIs which takes the list